style.scss

```diff
@@ -11,144 +12,270 @@ body {
+            }
 
-.tel {
-    font-size: 50px;
-    font-weight: bold;
-    margin: 15px;
-}
+            @media (min-width: $screen-md) {
+                max-width: 100px;
+            }
+        }
 
-.contacts {
-    background-color: $secondary-background;
-    display: flex;
-    justify-content: center;
-    align-items: center;
-}
+        .title {
+            color: $primary;
+            margin: 0;
+            font-weight: 700;
+            font-size: 2rem;
 
-.icon {
-    width: 50px;
-    height: 50px;
-}
+            @media (min-width: $screen-md) {
+                font-size: 3rem;
+            }
+        }
+    }
 
-.social-link {
-    margin: 5px;
-}
+    .contacts {
+        background-color: $secondary;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        padding: 1rem;
+        flex-direction: column;
+        gap: 1rem;
 
-.work {
-    background-image: url("./img/man.jpg");
-    background-size: cover;
-    background-position: center;
-    height: 600px;
-}
+        @media (min-width: $screen-sm) {
+            flex-direction: row;
+            gap: 3rem;
+        }
 
-.text-about-us {
-    line-height: 2em;
-    text-align: center;
-    font-size: 25px;
-    font-weight: bold;
-    padding-top: 100px;
-    margin: 0 800px 0 100px;
-}
+        .tel {
+            font-weight: 700;
+            margin: 0;
+            font-size: 1.5rem;
 
-.man-hand {
-    margin-top: 50px;
-}
+            @media (min-width: $screen-xs) {
+                font-size: 2rem;
+            }
 
-.work-for-you {
-    text-align: center;
-    color: $primary;
-    margin-left: 80px;
-}
+            @media (min-width: $screen-sm) {
+                font-size: 3rem;
+            }
 
-.container {
-    display: flex;
-    flex-wrap: wrap;
-    margin-left: 100px;
-    margin-bottom: 30px;
-    width: 100%;
-}
+            @media (min-width: $screen-md) {
+                font-size: 4rem;
+            }
+        }
 
-.container-with-text {
-    background-color: $primary-lighter;
-    border-radius: 8px;
-    padding: 20px 0;
-    border: 3px solid $light-background;
-    box-shadow: 0 5px 5px $color-border;
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-    margin: 20px;
-    width: 30%;
-}
+        .social-links-container {
+            display: flex;
+            gap: 1rem;
 
-.text {
-    font-size: 30px;
-    color: $light-background;
-    padding-bottom: 30px;
-    text-transform: uppercase;
-    line-height: 20px;
-    text-align: center;
-    font-weight: 900;
-    margin: 10px;
-    letter-spacing: 1.5px;
+            .social-link {
+                .icon {
+                    width: 50px;
+                    height: 50px;
+                }
+            }
+        }
+    }
 }
 
-.list {
-    line-height: 1.8;
-    align-self: flex-start;
-    list-style-type: none;
-    padding: 0;
-    margin: 20px;
-    font-size: 20px;
+main {
+    flex-grow: 1;
+
+    .work {
```

Comment:
2. This part must be refactored as well, but in the separate task. I'll create an issue for it.